src/components/RatioDisplay.js

```diff
@@ -6,23 +6,25 @@ import { Text } from '@makerdao/ui-components-core';
 const CDP_SAFETY_COLOR_PALETTE = {
   [CDP_SAFETY_LEVELS.DANGER]: getColor('orange.500'),
   [CDP_SAFETY_LEVELS.NEUTRAL]: getColor('grey.300'),
-  [CDP_SAFETY_LEVELS.SAFE]: getColor('teal.500')
+  [CDP_SAFETY_LEVELS.SAFE]: getColor('teal.500'),
+  neutral_inverse: getColor('steel')
 };
 
-function lookupCDPSafetyLevel(ratio) {
+function lookupCDPSafetyLevel(ratio, inverse) {
   ratio = parseFloat(ratio);
   if (ratio < 250) return CDP_SAFETY_LEVELS.DANGER;
-  if (ratio < 500) return CDP_SAFETY_LEVELS.NEUTRAL;
+  if (ratio < 500)
+    return inverse ? 'neutral_inverse' : CDP_SAFETY_LEVELS.NEUTRAL;
   return CDP_SAFETY_LEVELS.SAFE;
 }
 
-export default function RatioDisplay({ ratio, active }) {
+export default function RatioDisplay({ ratio, active, inverse, t }) {
   if (!ratio) return null;
-  const safetyLevel = lookupCDPSafetyLevel(ratio);
+  const safetyLevel = lookupCDPSafetyLevel(ratio, inverse);
 
   return (
     <Text
-      t="p6"
+      t={t ? t : 'p6'}
```

Comment:
The table component text defaults to "p3" and this is a workaround to fix that. Maybe it's bit hacky but let me know what you think